packages/terra-image/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "terra-image",
+  "main": "lib/Image.scss",
+  "private": true,
+  "version": "0.1.0",
+  "description": "terra-image",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-ui.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-image",
+    "Image",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-ui/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-ui#readme",
+  "devDependencies": {
+    "terra-toolkit": "^0.1.0"
+  },
+  "peerDependencies": {
+    "react": "^15.4.2",
```

Comment:
Lets remove `react` and `react-dom` from the peerDependencies.
Also, we need to add `terra-mixins` to the peerDependencies and dependencies.

e.g. 
```
"peerDependencies": {
    "terra-mixins": "1.0.0"
  },
  "dependencies": {
    "terra-mixins": "1.0.0"
  },
```